sale_order_report_with_note/README.rst

```diff
@@ -0,0 +1,60 @@
+.. image:: https://img.shields.io/badge/license-AGPLv3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl.html
+   :alt: License: AGPL-3
+
+=============================
+Sale Order Report With Note
+=============================
+
+* This module replaces the webkit Quotation / Sale Order report by
```

Comment:
Not sure if this description is correct as the report already exists in qweb. Actually it adds the notes to the existing report